src/ui/generic-components/CustomRecorderControl.swift

```diff
@@ -74,10 +74,15 @@ class CustomRecorderControl: RecorderControl, RecorderControlDelegate {
             }
             return $0.shortcut.keyCode == shortcut.keyCode && $0.shortcut.modifierFlags == shortcut.modifierFlags
         }) {
-            let existing = ControlsTab.shortcutControls[shortcutAlreadyAssigned.id]!
             let alert = NSAlert()
             alert.alertStyle = .critical
             alert.messageText = NSLocalizedString("Conflicting shortcut", comment: "")
+            if shortcutAlreadyAssigned.id.starts(with: "cycleSelectionLeftVim") {
+                alert.informativeText = String(format: NSLocalizedString("Shortcut already assigned in window selection with Vim keys.", comment: ""))
+                alert.runModal()
+                return
+            }
+            let existing = ControlsTab.shortcutControls[shortcutAlreadyAssigned.id]!
```

Comment:
I was able to crash the app by:

* Activating `vim keys` in the preferences
* Clicking to change `Select previous window` and picking `K`

It then crashed as `ControlsTab.shortcutControls[shortcutAlreadyAssigned.id]` was `nil`

I think there may be issues in the lifecycles of shortcut bindings.

Reply:
@lwouis I believe the arrow key crash predates this PR. I was able to reproduce it using https://github.com/lwouis/alt-tab-macos/commit/f5fc5ba8131fb65a2e54cf5c1795626b5118c1c4.